src/validate.ts

```diff
@@ -220,22 +220,21 @@ export const validate = async (
           // Make sure the token is verified on Etherscan.
           // Etherscan API is heavily rate limited, so sleep for 1s to avoid errors.
           await sleep(1000)
-          const { qResult } = await (
+          const { result: etherscanResult } = await (
```

Comment:
places like here we should be .catch(e => {
  console.error('unble to fetch etherscan', e, {chain, urlSarchParams}
}

Making it maximally easy to debug